### PR TITLE
add metrics selector for trigger form

### PR DESCRIPTION
**Pull request by GarageInc**
**Summary**

New signals by metrics

**Screenshots**
image
image

### CODE REVIEW

File: src/ducks/Signals/signalFormManager/signalCrudForm/formParts/metricTypes/MetricsList.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import cx from 'classnames'
+import Icon from '@santiment-network/ui/Icon'
+import { NO_GROUP } from '../../../../../Studio/Sidebar/utils'
+import styles from './MetricsList.module.scss'
+import { Description } from '../../../../../dataHub/metrics/descriptions'
+import HelpTooltip from '../../../../../../components/WatchlistOverview/WatchlistAnomalies/HelpTooltip'
```
Comment (DmitriVanGuard) [inline src/ducks/Signals/signalFormManager/signalCrudForm/formParts/metricTypes/MetricsList.js]: Incorrect imports order

File: src/ducks/Signals/signalFormManager/signalCrudForm/formParts/metricTypes/MetricsList.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import cx from 'classnames'
+import Icon from '@santiment-network/ui/Icon'
+import { NO_GROUP } from '../../../../../Studio/Sidebar/utils'
```
Comment (DmitriVanGuard) [inline src/ducks/Signals/signalFormManager/signalCrudForm/formParts/metricTypes/MetricsList.js]: I guess, much simpler would be to create a constant in this file directly

File: src/ducks/Signals/signalFormManager/signalCrudForm/formParts/metricTypes/TriggerFormMetricTypes.js [hunk 3]
```diff
@@ -77,22 +84,51 @@ export const TriggerFormMetricTypes = ({
...
+              iconPosition='left'
+              placeholder='Search for a Metric'
+              toggleMetric={onSelectMetric}
+              className={metricStyles.search}
+              categories={categories}
+            />
+
+            <div className={metricStyles.metrics}>
+              {categoriesKeys.map(key => {
+                return (
```
Comment (DmitriVanGuard) [inline src/ducks/Signals/signalFormManager/signalCrudForm/formParts/metricTypes/TriggerFormMetricTypes.js]: `return` is redundant

File: src/components/AutoresizeTextarea.js [hunk 4]
```diff
@@ -20,7 +20,7 @@ class AutoresizeTextarea extends Component {
   }
 
   state = {
-    value: (this.props.value || this.props.defaultValue).trim()
+    value: (this.props.value || this.props.defaultValue || '').trim()
```
[the remark this answers is not in the document]
Comment (GarageInc) [reply src/components/AutoresizeTextarea.js]: It caused a error, when input props 'value' has 'undefined' value

File: src/assets/signals/daa.svg [hunk 5]
```diff
@@ -1 +1 @@
...
...24 2.647 10.729c3.256-4.795 6.519.958 8.943 1.672 2.424.714 7.63-1.757 9.814 0 2.185 1.757.687 7.348-2.704 7.091-3.392-.256-.406 6.47-4.577 8.675z" fill="#E7EAF3"/><circle cx="10.008" cy="12.8" r="7.663" stroke="#505573" stroke-width="1.5"/><path fill-rule="evenodd" clip-rule="evenodd" d="M18.377 13.66a8.498 8.498 0 0 0 .019-1.5h6.823v-1.118a2.5 2.5 0 0 0-2.5-2.5h-5.455a8.436 8.436 0 0 0-1.123-1.5h6.578a4 4 0 0 1 4 4V21.767a4 4 0 0 1-4 4H8.249a4 4 0 0 1-4-4v-2.833c.455.427.958.805 1.5 1.123v1.71a2.5 2.5 0 0 0 2.5 2.5h14.47a2.5 2.5 0 0 0 2.5-2.5v-8.108h-6.842z" fill="#505573"/><path fill-rule="evenodd" clip-rule="evenodd" d="M14.575 10.444c0 .238-.112.45-.285.588l-2.24 1.793a.75.75 0 0 1-.938-1.172l.575-.46H6.195a.75.75 0 1 1 0-1.5h5.492l-.575-.459a.75.75 0 0 1 .937-1.17l2.237 1.789a.749.749 0 0 1 .289.59zM5.73 14.566a.748.748 0 0 0-.004 1.18l2.242 1.792a.75.75 0 0 0 .937-1.172l-.575-.46h5.492a.75.75 0 0 0 0-1.5H8.33l.574-.459a.75.75 0 0 0-.937-1.171l-2.237 1.79z" fill="#505573"/><path d="M19.23 21.242h3.744" stroke="#505573" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/></g><defs><clipPath id="clip0"><path fill="#fff" d="M0 0h28v30H0z"/></clipPath></defs></svg>
```
Comment (haritonasty) [inline src/assets/signals/daa.svg]: hm, how active svg could be green on screenshot, if you don't use it as a component?

Comment (GarageInc) [reply src/assets/signals/daa.svg]: Will change to component(without imports of svg)

**Pull request merged**